docs/src/docs/guielement/guielement-creation.adoc

```diff
@@ -29,6 +31,25 @@ GuiElement lower = upper.getSubElement(By.xpath("./p[@id='element']")); <3>
 <2> Corrects the selector prefix to './/'
 <3> Find any matching child
 
+== Element lists
+
+A locator can always return multiple elements. Even if a GuiElement is just a pointer to the first element of the result, it could contain more than one.
+
+You can get the amount of found elements by calling.
+[source,java]
+----
+GuiElement span = new GuiElement(driver, By.xpath("//a/span"));
+int count = span.getNumberOfFoundElements();
+----
+
+And access the elements by using the `getList()` method.
+[source,java]
+----
+List<GuiElement> allSpans = span.getList();
+----
+
+From this moment, every GuiElement of the list is now a pointer to the exact index position of the element results and behaves as the selector would by like `//a/span[n]`.
```

Comment:
Is the mentioned xpath correct for indexing? I think the xpath needs to be grouped: '(//a/span)[n]'

Reply:
Yes